packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -113,6 +112,8 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
   protected static getBaseExpressionContext(context: IActionContext): IBaseExpressionContext {
     const now: Date | undefined = context.get(KeysInitQuery.queryTimestamp);
     const baseIRI: string | undefined = context.get(KeysInitQuery.baseIRI);
+    context.setDefault(KeysInitQuery.overloadCache, {});
```

Comment:
We should also set a default in ActorInitQuery, to not require the user to define this context entry manually every time.

Reply:
Also, no need to set default here, we can probably inline the default value on the line below. (either using getDefault if that exists, or using `||`)

Reply:
Still there: https://github.com/comunica/comunica/blob/master/engines/query-sparql/test/QuerySparql-test.ts#L381-L385